experimental/traffic-portal/src/app/core/currentuser/currentuser.component.html

```diff
@@ -11,67 +11,71 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
-<tp-header [title]="currentUser ? currentUser.username : ''"></tp-header>
+<mat-card>
+	<main *ngIf="currentUser && !editMode">
+		<h1><span *ngIf=currentUser.fullName>{{currentUser.fullName}}, </span>{{currentUser.roleName}} user</h1>
+		<address>
+			<p *ngIf="currentUser.email || currentUser.phoneNumber"><a *ngIf=currentUser.email href="mailto:{{currentUser.email}}">{{currentUser.email}}</a><span *ngIf="currentUser.email && currentUser.phoneNumber">, </span><a *ngIf="currentUser.phoneNumber" href="tel:{{currentUser.phoneNumber}}">{{currentUser.phoneNumber}}</a></p>
+			<p *ngIf=currentUser.addressLine1>{{currentUser.addressLine1}}</p>
+			<p *ngIf=currentUser.addressLine2>{{currentUser.addressLine2}}</p>
+			<p *ngIf=currentUser.company>{{currentUser.company}}</p>
+			<p *ngIf="currentUser.city || currentUser.country || currentUser.stateOrProvince || currentUser.postalCode"><span *ngIf="currentUser.city">{{currentUser.city}}<span *ngIf="currentUser.country || currentUser.stateOrProvince || currentUser.postalCode">&nbsp;</span></span><span *ngIf="currentUser.stateOrProvince">{{currentUser.stateOrProvince}}<span *ngIf="currentUser.country || currentUser.postalCode">, </span></span><span *ngIf="currentUser.country">{{currentUser.country}}<span *ngIf="currentUser.postalCode">, </span></span><span *ngIf=currentUser.postalCode>{{currentUser.postalCode}}</span></p>
+		</address>
+		<div>
+			<h2>User Theme</h2>
+			<button mat-button (click)="themeSvc.loadTheme(theme)" *ngFor="let theme of themeSvc.themes">{{theme.name}}</button>
```

Comment:
The default type of buttons is `submit` - this should be a `button[type=button]` because it doesn't submit a form it just performs an interactive action.